cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/containers/NotifyContextRequestNGSIv2.java

```diff
@@ -0,0 +1,310 @@
+            return type;
+        }
+
+        /**
+         * Sets type.
+         *
+         * @param type the type
+         */
+        public void setType(String type) {
+            this.type = type;
+        }
+
+        /**
+         * Gets atributes.
+         *
+         * @return the atributes
+         */
+        public Map<String, Attribute> getAtributes() {
+            return attributes;
+        }
+
+        /**
+         * Sets atributes.
+         *
+         * @param atributes the atributes
+         */
+        public void setAtributes(Map<String, Attribute> atributes) {
+            this.attributes = atributes;
+        }
+
+        /**
+         * Data to context element notify context request . context element.
+         *
+         * @return the notify context request . context element
+         */
+        public NotifyContextRequest.ContextElement dataToContextElement () {
+            NotifyContextRequest.ContextElement contextElement = new NotifyContextRequest.ContextElement();
+            contextElement.setId(this.id);
+            contextElement.setType(this.type);
+            ArrayList<NotifyContextRequest.ContextAttribute> attributes = new ArrayList<NotifyContextRequest.ContextAttribute>();
+            for(Map.Entry<String, Attribute> map : this.attributes.entrySet()) {
+                attributes.add(map.getValue().attributeToContextAttribute(map.getKey()));
+            }
+            contextElement.setAttributes(attributes);
+                return contextElement;
+        }
+    }
+
+    /**
+     * The type Attribute.
+     */
+    public static class Attribute {
+        private JsonElement value;
+        private String type;
+        private Map<String, Metadata> metadata;
+
+        /**
+         * Gets value.
+         *
+         * @return the value
+         */
+        public JsonElement getValue() {
+            return value;
+        }
+
+        /**
+         * Sets value.
+         *
+         * @param value the value
+         */
+        public void setValue(JsonElement value) {
+            this.value = value;
+        }
+
+        /**
+         * Gets type.
+         *
+         * @return the type
+         */
+        public String getType() {
+            return type;
+        }
+
+        /**
+         * Sets type.
+         *
+         * @param type the type
+         */
+        public void setType(String type) {
+            this.type = type;
+        }
+
+        /**
+         * Gets metadata.
+         *
+         * @return the metadata
+         */
+        public Map<String, Metadata> getMetadata() {
+            return metadata;
+        }
+
+        /**
+         * Sets metadata.
+         *
+         * @param metadata the metadata
+         */
+        public void setMetadata(Map<String, Metadata> metadata) {
+            this.metadata = metadata;
+        }
+
+        /**
+         * Attribute to context attribute notify context request . context attribute.
```

Comment:
JavaDoc is a bit cryptic sometimes. Please do a global review.

Reply:
Fixed in [42ed891](https://github.com/telefonicaid/fiware-cygnus/pull/1762/commits/42ed8911fe2862c9ee59032f4d92ca6099f70ff9)